compiler/resolution/wrappers.cpp

```diff
@@ -577,12 +576,6 @@ static void addArgCoercion(FnSymbol*  fn,
       // Don't insert a readFE or readFF when deleting a sync/single.
```

Comment:
I think you’re right and think it’s worth looking at now.


Testing :-

Mike


On Aug 15, 2016, at 4:26 PM, Ben Harshbarger <notifications@github.com<mailto:notifications@github.com>> wrote:


In compiler/resolution/wrappers.cpp<https://github.com/chapel-lang/chapel/pull/4341#discussion_r74853812>:

> @@ -577,12 +576,6 @@ static void addArgCoercion(FnSymbol*  fn,
>        // Don't insert a readFE or readFF when deleting a sync/single.


In this branch that looks for a function named "free": It looks like its trying to avoid deleting the sync. I don't recall ever seeing a "free" function lying around, could it be that this branch is unused?

We don't need to address it in this PR, but something I was wondering about.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/chapel-lang/chapel/pull/4341/files/105d74f4510c9cbff1f4976d30bfd5a4cf70900d#r74853812>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACERphsLKJnWOS5x39YARx6tLAPJ-WZNks5qgPWNgaJpZM4Jk2tJ>.